### PR TITLE
Adding Avatar

### PR DESCRIPTION
**Pull request by chahatbansal**
Added my avatar to the website.

Here is the link: https://chahatbansal.github.io/gci17.fossasia.org/

### CODE REVIEW

**Review by sansyrox — CHANGES_REQUESTED** [review]
@chahatbansal please squash your commits , and try to make a short meaningful commit.
Also, set a maximum height and width as your image is covering too much space.

**Comment by chahatbansal** [comment]
@stealthanthrax Please specify height and width.
The image I uploaded is 1:1 ratio as mentioned.

**Comment by sansyrox** [comment]
@chahatbansal , try having a height of 120px or using bootstrap class of "col-2".

File: index.html [hunk 1]
```diff
@@ -287,7 +287,8 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        <div class="container">
 		            <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img alt="Pic" src="img/avatar1.png">
+					  <div class="col-2">
```
Comment (abhinavk96) [inline index.html:290]: @chahatbansal Please maintain the indentation

**Review by abhinavk96 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (abhinavk96) [reply index.html:290]: Also squash your commits :)

**Pull request closed without merging**